recipes/ncbi-vdb/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.9.0" %}
+{% set sha256 = "b4099e2fc3349eaf487219fbe798b22124949c89ffa1e7e6fbaa73a5178c8aff" %}
+
+package:
+  name: ncbi-vdb
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  fn: ncbi-vdb-{{ version }}.tar.gz
+  url: https://github.com/ncbi/ncbi-vdb/archive/{{ version }}-1.tar.gz
+  sha256: {{ sha256 }}
+
+
+requirements:
+  build:
+    - gcc # [not osx]
+    - llvm # [osx]
+    - perl
+    - libiconv
+    - libxml2
+    - ncbi-ngs-sdk
+    - bison
+
+  run:
+    #
+
+about:
+  home: https://github.com/ncbi/ncbi-vdb
+  license: Public Domain
+  summary: The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.
```

Comment:
I know that's what it says on their Github page. What does `ncbi-vdb` do though? 

```
summary: NCBI libraries for accessing the INSDC Sequence Read Archive (SRA)
description: |
  The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using
  data in the INSDC Sequence Read Archives. ...
```

Reply:
VDB is the database engine that all SRA tools use. SRA is built upon VDB.
VDB is a columnar database system with a number of unique features, but basically all SRA objects are stored in VDB.